tests/sles4sap/trento/test_hana_unregister_node.pm

```diff
@@ -0,0 +1,87 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Trento unregister the secondary node
+# Maintainer: QE-SAP <qe-sap@suse.de>, Michele Pagot <michele.pagot@suse.com>
+
+use strict;
+use warnings;
+use Mojo::Base 'publiccloud::basetest';
+use base 'consoletest';
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use utils 'script_retry';
+use qesapdeployment;
+use trento;
+
+sub run {
+    my ($self) = @_;
+    select_serial_terminal;
+
+    my $secondary_host = 'vmhana02';
+    cluster_print_cluster_status($secondary_host);
+
+    # Set secondary node in maintenance mode
+    my $prov = get_required_var('PUBLIC_CLOUD_PROVIDER');
+    qesap_ansible_cmd(cmd => "sudo crm configure property maintenance-mode=true",
+        provider => $prov,
+        filter => $secondary_host);
+    cluster_wait_status_by_regex($secondary_host, 900, m/global.+true/, "true");
+
+    # Stop hana instance
+    cluster_hdbadm($secondary_host, join(' ', 'HDB', 'stop'));
+    cluster_wait_status_by_regex($secondary_host, 900, m/miky\s+SFAIL/, "SFAIL");
```

Comment:
Here's another instance, this time with `SFAIL`.

Reply:
refactored